src/status_im2/contexts/chat/messages/list/events.cljs

```diff
@@ -36,7 +37,8 @@
        :type              :message
        :message-id        message-id
        :outgoing          (boolean outgoing)
-       :albumize?         albumize?}
+       ;; always albumize outgoing (image already loaded) messages
+       :albumize?         (if (and album-id outgoing) true albumize?)}
```

Comment:
I don't think we should do it that way. Each image is a separate message. Images in an album have different sizes depending on how many images are there. So for example if we are sending 6 images, the image sizes would be changing (becoming smaller) as more images are sent.

Reply:
So I see that in the app when sending multiple images, they do not seem to get sent one by one, but actually, they all load at once. I was expecting them to be sent one by one. Shouldn't they be sent one by one @cammellos ? And if so, they shouldn't be grouped initially to avoid the above scenaria?

Reply:
@OmarBasem they are sent one by one over the wire, that's correct, but that's opaque to the client, since it makes a single RPC call, which returns all the messages that have been sent, so from the client/sender perspective, it's like they are all sent together (there's some complications to this, but it's more to do whether one for example fails to send, but that happens in the background async).

From the sender perspective, they can be grouped immediately, since we know which one are going to be sent from the start, while the receiver will receive them one by one (same behavior as whatsapp).

Reply:
Thanks for the clarification @cammellos